students/KennanY/Lesson4/trigram.py

```diff
@@ -0,0 +1,44 @@
+import string
+
+trigramdata={}
+
+def readfile():
+    '''Open a file and reads text'''
+    with open("sherlock.txt", 'r') as file:
+        return (file.read().replace("\n", " ").lower())
+
+def print_trigram():
+    '''Prints the trigram on the screen'''
+    print('\nSherlock Trigram\n')
+    print('[Key] ---------> [Value]\n')
+    for key, value in trigramdata.items():
+        print('[' + key + '] ---->' + '[' + value +']' )
+
+def create_trigram(wordlist):
+    '''Creates a trigram from a list'''
+    #First two words become the key, next becomes value
+    counter=0
+    key=' '
+    for each in wordlist:
+        counter=counter+1
+        if counter%3 ==0:
+            trigramdata[key] = each
+            key=''
+        else:
+            key=key.strip() + ' ' + each
```

Comment:
it doesn't appear that they key is ever associated to more than 1 word since `append` is never used to append to the key?

also, the assignment asks to create a new story from the trigrams dict.

take a look at my version of trigrams solution to get an idea:
https://github.com/natasha-aleksandrova/Self_Paced-Online/blob/master/solutions/lesson4/trigrams.py